README.md

```diff
@@ -1,2 +1,55 @@
 # eslint-config-brightspace
+[![NPM version][npm-image]][npm-url]
+[![Build status][ci-image]][ci-url]
+[![Dependency Status][dependencies-image]][dependencies-url]
+
 Common Brightspace eslint configs.
+
+## Installation
+
+```shell
+npm install eslint-config-brightspace
+```
+
+## Usage
+
+Simply specify the `extends` property in the `.eslintrc` file as shown below. **Note:** omit the "eslint-config" when specifying the module, since eslint assumes it.
+
+### Default Config
+
+Specify the `extends` property in the `.eslintrc` file:
+
+```javascript
+{
+  "extends": "brightspace",
+  ...
+}
+```
+
+### React Config
+
+Specify the `react-config` config for the `extends` property: 
+
+```javascript
+{
+  "extends": "brightspace/react-config",
```

Comment:
Yeah - I actually was hoping this was not a requirement, and tried to use a module name that began with 'brightspace-' similar to a couple other repos/modules we have, but unfortunately it seems I had to follow suit with 'eslint-config-'.